src/memory/client/MessageBroker.ts

```diff
@@ -7,21 +7,37 @@
  *
  * SPDX-License-Identifier: EPL-2.0
  *********************************************************************/
-import { ClientRequest, ServerResponse, ReadMemory } from '../common/messages';
+import {
+    ClientRequest,
+    ServerResponse,
+    ReadMemory,
+    ChildDapNamesServerResponse,
+    GetChildDapNames,
+} from '../common/messages';
 
 declare function acquireVsCodeApi(): any;
 const vscode = acquireVsCodeApi();
 
 class MessageBroker {
     private currentToken = 1;
-    private queue: { [token: number]: (result: ServerResponse) => void } = {};
+    private queue1: { [token: number]: (result: ServerResponse) => void } = {};
```

Comment:
I don't understand why two queues now. Can you please explain it? It gets extra confusing to me because below you have two different event listeners installed on the same event too.